js/game.js

```diff
@@ -74,6 +77,14 @@ class Game {
             this.flaks.forEach(flak => flak.update());
             this.flaks = this.flaks.filter(flak => !flak.isDead);
         }
+        this.endGameUpdate();
+    }
+    endGameUpdate() {
+        if (!this.player.alive && this.deadTime < 5) {
+            this.deadTime++;
+            this.app.stage.filters = [this.colorMatrix];
```

Comment:
why are you passing this as a symbol? `[this.colorMatrix]` @BeanSparrow

Reply:
it's because a PIXI.ColorMatrixFilter is a literal matrix